### PR TITLE
update graphql acls

### PR DESCRIPTION
**Pull request by lechinoix**

### CODE REVIEW

File: hasura/metadata/databases/default/tables/public_payment_requests.yaml [hunk 1]
```diff
@@ -41,3 +41,29 @@ array_relationships:
         table:
           name: payments
           schema: public
+insert_permissions:
+  - role: project_lead
```
Comment (AnthonyBuisset) [inline hasura/metadata/databases/default/tables/public_payment_requests.yaml:45]: is it necessary to have a role project lead ?
I thought the project_leaded was enough

File: Makefile [hunk 2]
```diff
@@ -11,10 +11,23 @@ install:
...
+
+db/connect:
 	docker-compose up db -d
 	docker-compose exec -u postgres db psql marketplace_db
 
+db/update-staging-dump:
+	heroku pg:backups:capture --app onlydust-backend-staging-next
+	heroku pg:backups:download --app onlydust-backend-staging-next --output ./scripts/fixtures/latest.dump
+
+db/load-fixtures:
```
Comment (AnthonyBuisset) [inline Makefile:28]: ❤️

File: hasura/metadata/databases/default/tables/public_payment_requests.yaml [hunk 3]
```diff
@@ -31,3 +41,35 @@ array_relationships:
...
+        - requestor_id
+      filter:
+        project_id:
+          _in: X-Hasura-projects_leaded
+delete_permissions:
+  - role: project_lead
+    permission:
+      filter:
+        project_id:
+          _in: X-Hasura-projects_leaded
```
Comment (Bernardstanislas) [inline hasura/metadata/databases/default/tables/public_payment_requests.yaml:75]: ❤️

**Review by AnthonyBuisset — APPROVED** [review]

File: hasura/metadata/databases/default/tables/public_payment_requests.yaml [hunk 4]
```diff
@@ -31,3 +41,35 @@ array_relationships:
         table:
           name: payments
           schema: public
+insert_permissions:
+  - role: project_lead
+    permission:
+      check:
+        project_id:
+          _in: X-Hasura-projects_leaded
```
Comment (Bernardstanislas) [inline hasura/metadata/databases/default/tables/public_payment_requests.yaml:49]: Trop stylé

**Review by Bernardstanislas — APPROVED** [review]

File: hasura/metadata/databases/default/tables/public_payment_requests.yaml (hunk 1 above)
Comment (lechinoix) [reply hasura/metadata/databases/default/tables/public_payment_requests.yaml:45]: Yes it is not necessary at the moment, I'll add it on user role

**Pull request merged**